playwright/_impl/_connection.py

```diff
@@ -205,38 +206,17 @@ def local_utils(self) -> "LocalUtils":
     def mark_as_remote(self) -> None:
         self.is_remote = True
 
-    async def run_as_sync(self) -> None:
+    async def init_sync(self) -> None:
         self._is_sync = True
-        await self.run()
+        await self.init()
 
-    async def run(self) -> None:
-        self._loop = asyncio.get_running_loop()
+    async def init(self) -> None:
```

Comment:
This seems to be a sync method. Perhaps it could also be merged into the constructor.